cmd/example-mysql/main.go

```diff
@@ -87,3 +119,33 @@ func main() {
 		klog.Exitf("ListenAndServe: %v", err)
 	}
 }
+
+// parseTileLevelIndex takes level and index in string, validates and returns them in uint64.
+//
+// Examples:
+// "/tile/0/x001/x234/067" means level 0 and index 1234067 of a full tile.
+// "/tile/0/x001/x234/067.p/8" means level 0, index 1234067 and width 8 of a partial tile.
+func parseTileLevelIndex(level, index string) (uint64, uint64, error) {
+	l, err := strconv.ParseUint(level, 10, 64)
+	if err != nil {
+		return 0, 0, fmt.Errorf("malformed URL: failed to parse tile level")
+	}
+
+	var i uint64
+	switch splittedIndexPath := strings.Split(index, "/"); len(splittedIndexPath) {
```

Comment:
nit: splitted isn't a word because English is weird. `I split the path` can be the past tense or the present tense. 🙃 

`splitIndexPath` works I think? Or `indexParts` to shorten further.

Reply:
Ooops! Updated to `indexParts`.